logfire/variables/remote.py

```
@@ -188,6 +196,48 @@ def _log_error(self, message: str, exc: Exception) -> None:
+            message: The warning message.
+            exc: The exception that occurred.
+        """
+        if self._logfire is not None:
+            self._logfire.warn(
+                '{message}: {error_type}: {error}',
+                message=message,
+                error_type=type(exc).__name__,
+                error=str(exc),
+            )
+        else:
+            warnings.warn(f'{message}: {type(exc).__name__}: {exc}', category=RuntimeWarning)
```

Ruff (0.15.21)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)